object.sc

```diff
@@ -0,0 +1,6 @@
++ Object {
```

Comment:
You can do o.isKindOf(AbstractVector). Is this really needed ?

Reply:
Not anymore. I removed that line in Boids and can remove it here, too.